packages/types/lib/logs/messages.ts

```diff
@@ -43,9 +43,10 @@ export interface OperationAction {
     type: 'action';
     action: 'run';
 }
+// TODO: rename to OperationConnection
```

Comment:
`OperationConnection` and `type: connection` will  be better name but it would be a breaking change for the log search.